src/main/java/com/snowflake/kafka/connector/internal/SnowflakeTelemetryServiceV1.java

```diff
@@ -126,10 +136,7 @@ private void send(TelemetryType type, JsonNode data, boolean flush)
     {
       telemetry.addLogToBatch(TelemetryUtil.buildJobData(msg));
       logDebug("sending telemetry data: {}", data.toString());
-      if (flush)
-      {
-        telemetry.sendBatchAsync();
-      }
+      telemetry.sendBatchAsync();
```

Comment:
Removed this flush parameter since we always flush.

Reply:
Then why have the parameter ?

Reply:
Can you remove the parameter ?

Reply:
Yes, this flush parameter is already removed.